src/lib/CMakeLists.txt

```diff
@@ -33,6 +33,8 @@
 
 px4_add_git_submodule(TARGET git_matrix PATH "matrix")
 px4_add_git_submodule(TARGET git_monocypher PATH "crypto/monocypher")
+px4_add_git_submodule(TARGET git_libtomcrypt PATH "crypto/libtomcrypt")
```

Comment:
Move this into src/lib/crypto/CMakeLists.txt?

Reply:
Then if you limit the px4_add_git_submodule() call to either NuttX or a flag that enables crypto the submodule won't even populate for the current clang-tidy build (px4_sitl).

Reply:
Thanks, will try that!